src/osd/ReplicatedPG.cc

```diff
@@ -4351,8 +4352,8 @@ int ReplicatedPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
           if (result >= 0) {
 	    if (maybe_create_new_object(ctx)) {
               ctx->mod_desc.create();
+	      t->touch(soid);
```

Comment:
It's been a while, but IIRC I hit this while testing with the new Thrasher changes.  I can't remember if I recognized it as related to the Thrasher additions or just something I hit and found the fix and added it to the branch.